File: app/src/main/java/com/example/techjini/maps/GoogleAPIService.java
package com.example.techjini.maps;


import retrofit2.http.GET;
import retrofit2.http.Query;
import rx.Observable;

public interface GoogleAPIService {

    @GET("geocode/json")
    Observable<PlacesModel> getGeoCode(
            @Query("address") String addr,
            @Query("key") String googlekey);


    @GET("place/details/json")
    Observable<Place> getPlacesDetails(
            @Query("placeid") String placeid,
            @Query("key") String googlekey);
}

File: app/src/main/java/com/example/techjini/maps/GoogleAPIService.kt
package com.example.techjini.maps


import retrofit2.http.GET
import retrofit2.http.Query
import rx.Observable

interface GoogleAPIService {

    @GET("geocode/json")
    fun getGeoCode(
            @Query("address") addr: String,
            @Query("key") googlekey: String): Observable<PlacesModel>


    @GET("place/details/json")
    fun getPlacesDetails(
            @Query("placeid") placeid: String,
            @Query("key") googlekey: String): Observable<Place>
}
